src/acl/Node.h

```diff
@@ -67,7 +73,12 @@ class Node
     /// printed parameters are collected from all same-name "acl" directives.
     void dumpWhole(const char *directiveName, std::ostream &);
 
-    char name[ACL_NAME_SZ];
```

Comment:
Please check for other uses of `ACL_NAME_SZ` limitation and remove the macro as part of this.

Reply:
Done to avoid arguing (commit cb4a4bb).